diff --git a/idioms/nixos/kernel/before.nix b/idioms/nixos/kernel/after-nixfmt.nix
index d147155..fa81fe0 100644
--- a/idioms/nixos/kernel/before.nix
+++ b/idioms/nixos/kernel/after-nixfmt.nix
@@ -8,21 +8,18 @@ let
   inherit (config.boot.kernel) features randstructSeed;
   inherit (config.boot.kernelPackages) kernel;
 
-  kernelModulesConf = pkgs.writeText "nixos.conf"
-    ''
-      ${concatStringsSep "\n" config.boot.kernelModules}
-    '';
+  kernelModulesConf = pkgs.writeText "nixos.conf" ''
+    ${concatStringsSep "\n" config.boot.kernelModules}
+  '';
 
-in
-
-{
+in {
 
   ###### interface
 
   options = {
 
     boot.kernel.features = mkOption {
-      default = {};
+      default = { };
       example = literalExpression "{ debug = true; }";
       internal = true;
       description = ''
@@ -37,13 +34,15 @@ in
     boot.kernelPackages = mkOption {
       default = pkgs.linuxPackages;
       type = types.unspecified // { merge = mergeEqualOption; };
-      apply = kernelPackages: kernelPackages.extend (self: super: {
-        kernel = super.kernel.override (originalArgs: {
-          inherit randstructSeed;
-          kernelPatches = (originalArgs.kernelPatches or []) ++ kernelPatches;
-          features = lib.recursiveUpdate super.kernel.features features;
+      apply = kernelPackages:
+        kernelPackages.extend (self: super: {
+          kernel = super.kernel.override (originalArgs: {
+            inherit randstructSeed;
+            kernelPatches = (originalArgs.kernelPatches or [ ])
+              ++ kernelPatches;
+            features = lib.recursiveUpdate super.kernel.features features;
+          });
         });
-      });
       # We don't want to evaluate all of linuxPackages for the manual
       # - some of it might not even evaluate correctly.
       defaultText = literalExpression "pkgs.linuxPackages";
@@ -64,7 +63,7 @@ in
 
     boot.kernelPatches = mkOption {
       type = types.listOf types.attrs;
-      default = [];
+      default = [ ];
       example = literalExpression "[ pkgs.kernelPatches.ubuntu_fan_4_4 ]";
       description = "A list of additional patches to apply to the kernel.";
     };
@@ -115,14 +114,14 @@ in
 
     boot.extraModulePackages = mkOption {
       type = types.listOf types.package;
-      default = [];
+      default = [ ];
       example = literalExpression "[ config.boot.kernelPackages.nvidia_x11 ]";
       description = "A list of additional packages supplying kernel modules.";
     };
 
     boot.kernelModules = mkOption {
       type = types.listOf types.str;
-      default = [];
+      default = [ ];
       description = ''
         The set of kernel modules to be loaded in the second stage of
         the boot process.  Note that modules that are needed to
@@ -134,7 +133,7 @@ in
 
     boot.initrd.availableKernelModules = mkOption {
       type = types.listOf types.str;
-      default = [];
+      default = [ ];
       example = [ "sata_nv" "ext3" ];
       description = ''
         The set of kernel modules in the initial ramdisk used during the
@@ -155,7 +154,7 @@ in
 
     boot.initrd.kernelModules = mkOption {
       type = types.listOf types.str;
-      default = [];
+      default = [ ];
       description = "List of modules that are always loaded by the initrd.";
     };
 
@@ -172,7 +171,7 @@ in
     system.modulesTree = mkOption {
       type = types.listOf types.path;
       internal = true;
-      default = [];
+      default = [ ];
       description = ''
         Tree of kernel modules.  This includes the kernel, plus modules
         built outside of the kernel.  Combine these into a single tree of
@@ -183,7 +182,7 @@ in
     };
 
     system.requiredKernelConfig = mkOption {
-      default = [];
+      default = [ ];
       example = literalExpression ''
         with config.lib.kernelConfig; [
           (isYes "MODULES")
@@ -202,149 +201,158 @@ in
 
   };
 
-
   ###### implementation
 
-  config = mkMerge
-    [ (mkIf config.boot.initrd.enable {
-        boot.initrd.availableKernelModules =
-          optionals config.boot.initrd.includeDefaultModules ([
-            # Note: most of these (especially the SATA/PATA modules)
-            # shouldn't be included by default since nixos-generate-config
-            # detects them, but I'm keeping them for now for backwards
-            # compatibility.
-
-            # Some SATA/PATA stuff.
-            "ahci"
-            "sata_nv"
-            "sata_via"
-            "sata_sis"
-            "sata_uli"
-            "ata_piix"
-            "pata_marvell"
-
-            # Standard SCSI stuff.
-            "sd_mod"
-            "sr_mod"
-
-            # SD cards and internal eMMC drives.
-            "mmc_block"
-
-            # Support USB keyboards, in case the boot fails and we only have
-            # a USB keyboard, or for LUKS passphrase prompt.
-            "uhci_hcd"
-            "ehci_hcd"
-            "ehci_pci"
-            "ohci_hcd"
-            "ohci_pci"
-            "xhci_hcd"
-            "xhci_pci"
-            "usbhid"
-            "hid_generic" "hid_lenovo" "hid_apple" "hid_roccat"
-            "hid_logitech_hidpp" "hid_logitech_dj" "hid_microsoft"
-
-          ] ++ optionals pkgs.stdenv.hostPlatform.isx86 [
-            # Misc. x86 keyboard stuff.
-            "pcips2" "atkbd" "i8042"
-
-            # x86 RTC needed by the stage 2 init script.
-            "rtc_cmos"
-          ]);
-
-        boot.initrd.kernelModules =
-          optionals config.boot.initrd.includeDefaultModules [
-            # For LVM.
-            "dm_mod"
-          ];
-      })
-
-      (mkIf (!config.boot.isContainer) {
-        system.build = { inherit kernel; };
-
-        system.modulesTree = [ kernel ] ++ config.boot.extraModulePackages;
-
-        # Implement consoleLogLevel both in early boot and using sysctl
-        # (so you don't need to reboot to have changes take effect).
-        boot.kernelParams =
-          [ "loglevel=${toString config.boot.consoleLogLevel}" ] ++
-          optionals config.boot.vesa [ "vga=0x317" "nomodeset" ];
-
-        boot.kernel.sysctl."kernel.printk" = mkDefault config.boot.consoleLogLevel;
-
-        boot.kernelModules = [ "loop" "atkbd" ];
-
-        # The Linux kernel >= 2.6.27 provides firmware.
-        hardware.firmware = [ kernel ];
-
-        # Create /etc/modules-load.d/nixos.conf, which is read by
-        # systemd-modules-load.service to load required kernel modules.
-        environment.etc =
-          { "modules-load.d/nixos.conf".source = kernelModulesConf;
-          };
-
-        systemd.services.systemd-modules-load =
-          { wantedBy = [ "multi-user.target" ];
-            restartTriggers = [ kernelModulesConf ];
-            serviceConfig =
-              { # Ignore failed module loads.  Typically some of the
-                # modules in ‘boot.kernelModules’ are "nice to have but
-                # not required" (e.g. acpi-cpufreq), so we don't want to
-                # barf on those.
-                SuccessExitStatus = "0 1";
-              };
-          };
-
-        lib.kernelConfig = {
-          isYes = option: {
-            assertion = config: config.isYes option;
-            message = "CONFIG_${option} is not yes!";
-            configLine = "CONFIG_${option}=y";
-          };
-
-          isNo = option: {
-            assertion = config: config.isNo option;
-            message = "CONFIG_${option} is not no!";
-            configLine = "CONFIG_${option}=n";
-          };
-
-          isModule = option: {
-            assertion = config: config.isModule option;
-            message = "CONFIG_${option} is not built as a module!";
-            configLine = "CONFIG_${option}=m";
-          };
-
-          ### Usually you will just want to use these two
-          # True if yes or module
-          isEnabled = option: {
-            assertion = config: config.isEnabled option;
-            message = "CONFIG_${option} is not enabled!";
-            configLine = "CONFIG_${option}=y";
-          };
-
-          # True if no or omitted
-          isDisabled = option: {
-            assertion = config: config.isDisabled option;
-            message = "CONFIG_${option} is not disabled!";
-            configLine = "CONFIG_${option}=n";
-          };
+  config = mkMerge [
+    (mkIf config.boot.initrd.enable {
+      boot.initrd.availableKernelModules =
+        optionals config.boot.initrd.includeDefaultModules ([
+          # Note: most of these (especially the SATA/PATA modules)
+          # shouldn't be included by default since nixos-generate-config
+          # detects them, but I'm keeping them for now for backwards
+          # compatibility.
+
+          # Some SATA/PATA stuff.
+          "ahci"
+          "sata_nv"
+          "sata_via"
+          "sata_sis"
+          "sata_uli"
+          "ata_piix"
+          "pata_marvell"
+
+          # Standard SCSI stuff.
+          "sd_mod"
+          "sr_mod"
+
+          # SD cards and internal eMMC drives.
+          "mmc_block"
+
+          # Support USB keyboards, in case the boot fails and we only have
+          # a USB keyboard, or for LUKS passphrase prompt.
+          "uhci_hcd"
+          "ehci_hcd"
+          "ehci_pci"
+          "ohci_hcd"
+          "ohci_pci"
+          "xhci_hcd"
+          "xhci_pci"
+          "usbhid"
+          "hid_generic"
+          "hid_lenovo"
+          "hid_apple"
+          "hid_roccat"
+          "hid_logitech_hidpp"
+          "hid_logitech_dj"
+          "hid_microsoft"
+
+        ] ++ optionals pkgs.stdenv.hostPlatform.isx86 [
+          # Misc. x86 keyboard stuff.
+          "pcips2"
+          "atkbd"
+          "i8042"
+
+          # x86 RTC needed by the stage 2 init script.
+          "rtc_cmos"
+        ]);
+
+      boot.initrd.kernelModules =
+        optionals config.boot.initrd.includeDefaultModules [
+          # For LVM.
+          "dm_mod"
+        ];
+    })
+
+    (mkIf (!config.boot.isContainer) {
+      system.build = { inherit kernel; };
+
+      system.modulesTree = [ kernel ] ++ config.boot.extraModulePackages;
+
+      # Implement consoleLogLevel both in early boot and using sysctl
+      # (so you don't need to reboot to have changes take effect).
+      boot.kernelParams = [ "loglevel=${toString config.boot.consoleLogLevel}" ]
+        ++ optionals config.boot.vesa [ "vga=0x317" "nomodeset" ];
+
+      boot.kernel.sysctl."kernel.printk" =
+        mkDefault config.boot.consoleLogLevel;
+
+      boot.kernelModules = [ "loop" "atkbd" ];
+
+      # The Linux kernel >= 2.6.27 provides firmware.
+      hardware.firmware = [ kernel ];
+
+      # Create /etc/modules-load.d/nixos.conf, which is read by
+      # systemd-modules-load.service to load required kernel modules.
+      environment.etc = {
+        "modules-load.d/nixos.conf".source = kernelModulesConf;
+      };
+
+      systemd.services.systemd-modules-load = {
+        wantedBy = [ "multi-user.target" ];
+        restartTriggers = [ kernelModulesConf ];
+        serviceConfig = { # Ignore failed module loads.  Typically some of the
+          # modules in ‘boot.kernelModules’ are "nice to have but
+          # not required" (e.g. acpi-cpufreq), so we don't want to
+          # barf on those.
+          SuccessExitStatus = "0 1";
         };
+      };
 
-        # The config options that all modules can depend upon
-        system.requiredKernelConfig = with config.lib.kernelConfig;
-          [
-            # !!! Should this really be needed?
-            (isYes "MODULES")
-            (isYes "BINFMT_ELF")
-          ] ++ (optional (randstructSeed != "") (isYes "GCC_PLUGIN_RANDSTRUCT"));
+      lib.kernelConfig = {
+        isYes = option: {
+          assertion = config: config.isYes option;
+          message = "CONFIG_${option} is not yes!";
+          configLine = "CONFIG_${option}=y";
+        };
 
-        # nixpkgs kernels are assumed to have all required features
-        assertions = if config.boot.kernelPackages.kernel ? features then [] else
-          let cfg = config.boot.kernelPackages.kernel.config; in map (attrs:
-            { assertion = attrs.assertion cfg; inherit (attrs) message; }
-          ) config.system.requiredKernelConfig;
+        isNo = option: {
+          assertion = config: config.isNo option;
+          message = "CONFIG_${option} is not no!";
+          configLine = "CONFIG_${option}=n";
+        };
 
-      })
+        isModule = option: {
+          assertion = config: config.isModule option;
+          message = "CONFIG_${option} is not built as a module!";
+          configLine = "CONFIG_${option}=m";
+        };
 
-    ];
+        ### Usually you will just want to use these two
+        # True if yes or module
+        isEnabled = option: {
+          assertion = config: config.isEnabled option;
+          message = "CONFIG_${option} is not enabled!";
+          configLine = "CONFIG_${option}=y";
+        };
+
+        # True if no or omitted
+        isDisabled = option: {
+          assertion = config: config.isDisabled option;
+          message = "CONFIG_${option} is not disabled!";
+          configLine = "CONFIG_${option}=n";
+        };
+      };
+
+      # The config options that all modules can depend upon
+      system.requiredKernelConfig = with config.lib.kernelConfig;
+        [
+          # !!! Should this really be needed?
+          (isYes "MODULES")
+          (isYes "BINFMT_ELF")
+        ] ++ (optional (randstructSeed != "") (isYes "GCC_PLUGIN_RANDSTRUCT"));
+
+      # nixpkgs kernels are assumed to have all required features
+      assertions = if config.boot.kernelPackages.kernel ? features then
+        [ ]
+      else
+        let cfg = config.boot.kernelPackages.kernel.config;
+        in map (attrs: {
+          assertion = attrs.assertion cfg;
+          inherit (attrs) message;
+        }) config.system.requiredKernelConfig;
+
+    })
+
+  ];
 
 }
