diff --git a/idioms/nixos/kernel/before.nix b/idioms/nixos/kernel/after-nixpkgs-fmt.nix
index d147155..440dd75 100644
--- a/idioms/nixos/kernel/before.nix
+++ b/idioms/nixos/kernel/after-nixpkgs-fmt.nix
@@ -24,3 +24,3 @@ in
     boot.kernel.features = mkOption {
-      default = {};
+      default = { };
       example = literalExpression "{ debug = true; }";
@@ -42,3 +42,3 @@ in
           inherit randstructSeed;
-          kernelPatches = (originalArgs.kernelPatches or []) ++ kernelPatches;
+          kernelPatches = (originalArgs.kernelPatches or [ ]) ++ kernelPatches;
           features = lib.recursiveUpdate super.kernel.features features;
@@ -66,3 +66,3 @@ in
       type = types.listOf types.attrs;
-      default = [];
+      default = [ ];
       example = literalExpression "[ pkgs.kernelPatches.ubuntu_fan_4_4 ]";
@@ -117,3 +117,3 @@ in
       type = types.listOf types.package;
-      default = [];
+      default = [ ];
       example = literalExpression "[ config.boot.kernelPackages.nvidia_x11 ]";
@@ -124,3 +124,3 @@ in
       type = types.listOf types.str;
-      default = [];
+      default = [ ];
       description = ''
@@ -136,3 +136,3 @@ in
       type = types.listOf types.str;
-      default = [];
+      default = [ ];
       example = [ "sata_nv" "ext3" ];
@@ -157,3 +157,3 @@ in
       type = types.listOf types.str;
-      default = [];
+      default = [ ];
       description = "List of modules that are always loaded by the initrd.";
@@ -174,3 +174,3 @@ in
       internal = true;
-      default = [];
+      default = [ ];
       description = ''
@@ -185,3 +185,3 @@ in
     system.requiredKernelConfig = mkOption {
-      default = [];
+      default = [ ];
       example = literalExpression ''
@@ -208,3 +208,4 @@ in
   config = mkMerge
-    [ (mkIf config.boot.initrd.enable {
+    [
+      (mkIf config.boot.initrd.enable {
         boot.initrd.availableKernelModules =
@@ -242,4 +243,9 @@ in
             "usbhid"
-            "hid_generic" "hid_lenovo" "hid_apple" "hid_roccat"
-            "hid_logitech_hidpp" "hid_logitech_dj" "hid_microsoft"
+            "hid_generic"
+            "hid_lenovo"
+            "hid_apple"
+            "hid_roccat"
+            "hid_logitech_hidpp"
+            "hid_logitech_dj"
+            "hid_microsoft"
 
@@ -247,3 +253,5 @@ in
             # Misc. x86 keyboard stuff.
-            "pcips2" "atkbd" "i8042"
+            "pcips2"
+            "atkbd"
+            "i8042"
 
@@ -281,3 +289,4 @@ in
         environment.etc =
-          { "modules-load.d/nixos.conf".source = kernelModulesConf;
+          {
+            "modules-load.d/nixos.conf".source = kernelModulesConf;
           };
@@ -285,6 +294,8 @@ in
         systemd.services.systemd-modules-load =
-          { wantedBy = [ "multi-user.target" ];
+          {
+            wantedBy = [ "multi-user.target" ];
             restartTriggers = [ kernelModulesConf ];
             serviceConfig =
-              { # Ignore failed module loads.  Typically some of the
+              {
+                # Ignore failed module loads.  Typically some of the
                 # modules in ‘boot.kernelModules’ are "nice to have but
@@ -340,6 +351,8 @@ in
         # nixpkgs kernels are assumed to have all required features
-        assertions = if config.boot.kernelPackages.kernel ? features then [] else
-          let cfg = config.boot.kernelPackages.kernel.config; in map (attrs:
+        assertions = if config.boot.kernelPackages.kernel ? features then [ ] else
+        let cfg = config.boot.kernelPackages.kernel.config; in map
+          (attrs:
             { assertion = attrs.assertion cfg; inherit (attrs) message; }
-          ) config.system.requiredKernelConfig;
+          )
+          config.system.requiredKernelConfig;
 
