diff --git a/discussions/function-signatures/after-nixpkgs-fmt.nix b/discussions/function-signatures/after-alejandra.nix
index 72e5e11..4299091 100644
--- a/discussions/function-signatures/after-nixpkgs-fmt.nix
+++ b/discussions/function-signatures/after-alejandra.nix
@@ -2,27 +2,42 @@
   # Short that was single-line
-  ({ nixpkgs, ... }@attrs: nixpkgs)
+  ({nixpkgs, ...} @ attrs: nixpkgs)
 
   # Short that was multi-line
-  ({ nixpkgs
-   , ...
-   }@attrs: nixpkgs)
+  ({nixpkgs, ...} @ attrs: nixpkgs)
 
   # Long that was single-line
-  (attrs@{ stdenv, lib, gcc, coreutils, gnufind, gnugrep, gnumake, gnutar, gnupg, gnused, gnuk, gnubg, ... }: stdenv)
+  (attrs @ {
+    stdenv,
+    lib,
+    gcc,
+    coreutils,
+    gnufind,
+    gnugrep,
+    gnumake,
+    gnutar,
+    gnupg,
+    gnused,
+    gnuk,
+    gnubg,
+    ...
+  }:
+    stdenv)
 
   # Long that was multi-line
-  (attrs@{ stdenv
-   , lib
-   , gcc
-   , coreutils
-   , gnufind
-   , gnugrep
-   , gnumake
-   , gnutar
-   , gnupg
-   , gnused
-   , gnuk
-   , gnubg
-   , ...
-   }: stdenv)
+  (attrs @ {
+    stdenv,
+    lib,
+    gcc,
+    coreutils,
+    gnufind,
+    gnugrep,
+    gnumake,
+    gnutar,
+    gnupg,
+    gnused,
+    gnuk,
+    gnubg,
+    ...
+  }:
+    stdenv)
 ]
