diff --git a/discussions/function-signatures/after-nixpkgs-fmt.nix b/discussions/function-signatures/after-nixfmt.nix
index 72e5e11..f209248 100644
--- a/discussions/function-signatures/after-nixpkgs-fmt.nix
+++ b/discussions/function-signatures/after-nixfmt.nix
@@ -5,24 +5,13 @@
   # Short that was multi-line
-  ({ nixpkgs
-   , ...
-   }@attrs: nixpkgs)
+  ({ nixpkgs, ... }@attrs: nixpkgs)
 
   # Long that was single-line
-  (attrs@{ stdenv, lib, gcc, coreutils, gnufind, gnugrep, gnumake, gnutar, gnupg, gnused, gnuk, gnubg, ... }: stdenv)
+  (attrs@{ stdenv, lib, gcc, coreutils, gnufind, gnugrep, gnumake, gnutar, gnupg
+    , gnused, gnuk, gnubg, ... }:
+    stdenv)
 
   # Long that was multi-line
-  (attrs@{ stdenv
-   , lib
-   , gcc
-   , coreutils
-   , gnufind
-   , gnugrep
-   , gnumake
-   , gnutar
-   , gnupg
-   , gnused
-   , gnuk
-   , gnubg
-   , ...
-   }: stdenv)
+  (attrs@{ stdenv, lib, gcc, coreutils, gnufind, gnugrep, gnumake, gnutar, gnupg
+    , gnused, gnuk, gnubg, ... }:
+    stdenv)
 ]
