diff --git a/discussions/if-then-else/before.nix b/discussions/if-then-else/after-nixfmt.nix
index 2bae882..14aeecb 100644
--- a/discussions/if-then-else/before.nix
+++ b/discussions/if-then-else/after-nixfmt.nix
@@ -2,15 +2,14 @@
   # small
-  (if cond
-  then foo
-  else bar)
+  (if cond then foo else bar)
 
   # nested
-  (if cond
-  then if
-  looooooooooooooooooooooooooooooooooooooooooooooooooooong
-  then foo
-  else bar
-  else if cond
-  then foo
-  else bar)
+  (if cond then
+    if looooooooooooooooooooooooooooooooooooooooooooooooooooong then
+      foo
+    else
+      bar
+  else if cond then
+    foo
+  else
+    bar)
 ]
