diff --git a/discussions/if-then-else/after-nixfmt.nix b/discussions/if-then-else/after-alejandra.nix
index 14aeecb..b479736 100644
--- a/discussions/if-then-else/after-nixfmt.nix
+++ b/discussions/if-then-else/after-alejandra.nix
@@ -2,14 +2,19 @@
   # small
-  (if cond then foo else bar)
+  (
+    if cond
+    then foo
+    else bar
+  )
 
   # nested
-  (if cond then
-    if looooooooooooooooooooooooooooooooooooooooooooooooooooong then
-      foo
-    else
-      bar
-  else if cond then
-    foo
-  else
-    bar)
+  (
+    if cond
+    then
+      if looooooooooooooooooooooooooooooooooooooooooooooooooooong
+      then foo
+      else bar
+    else if cond
+    then foo
+    else bar
+  )
 ]
