diff --git a/monsters/no-newlines/after-alejandra.nix b/monsters/no-newlines/after-nixfmt.nix
index 25ff805..daf77bf 100644
--- a/monsters/no-newlines/after-alejandra.nix
+++ b/monsters/no-newlines/after-nixfmt.nix
@@ -1,19 +1,4 @@
-{
-  stdenv,
-  lib,
-  fetchFromGitLab,
-  cairo,
-  desktop-file-utils,
-  gettext,
-  glib,
-  gtk4,
-  libadwaita,
-  meson,
-  ninja,
-  pango,
-  pkg-config,
-  python3,
-  rustPlatform,
-  wrapGAppsHook4,
-}:
+{ stdenv, lib, fetchFromGitLab, cairo, desktop-file-utils, gettext, glib, gtk4
+, libadwaita, meson, ninja, pango, pkg-config, python3, rustPlatform
+, wrapGAppsHook4 }:
 stdenv.mkDerivation rec {
@@ -47,3 +32,3 @@ stdenv.mkDerivation rec {
   ];
-  buildInputs = [cairo glib gtk4 libadwaita pango];
+  buildInputs = [ cairo glib gtk4 libadwaita pango ];
   postPatch = ''
@@ -55,6 +40,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
-    description = "Checks whether the contrast between two colors meet the WCAG requirements";
+    description =
+      "Checks whether the contrast between two colors meet the WCAG requirements";
     homepage = "https://gitlab.gnome.org/World/design/contrast";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [jtojnar];
+    maintainers = with maintainers; [ jtojnar ];
     platforms = platforms.unix;
