diff --git a/monsters/no-newlines/after-nixpkgs-fmt.nix b/monsters/no-newlines/after-nixfmt.nix
index 66cce27..daf77bf 100644
--- a/monsters/no-newlines/after-nixpkgs-fmt.nix
+++ b/monsters/no-newlines/after-nixfmt.nix
@@ -1,6 +1,20 @@
-{ stdenv, lib, fetchFromGitLab, cairo, desktop-file-utils, gettext, glib, gtk4, libadwaita, meson, ninja, pango, pkg-config, python3, rustPlatform, wrapGAppsHook4 }: stdenv.mkDerivation rec {
+{ stdenv, lib, fetchFromGitLab, cairo, desktop-file-utils, gettext, glib, gtk4
+, libadwaita, meson, ninja, pango, pkg-config, python3, rustPlatform
+, wrapGAppsHook4 }:
+stdenv.mkDerivation rec {
   pname = "contrast";
   version = "0.0.5";
-  src = fetchFromGitLab { domain = "gitlab.gnome.org"; group = "World"; owner = "design"; repo = "contrast"; rev = version; sha256 = "cypSbqLwSmauOoWOuppWpF3hvrxiqmkLspxAWzvlUC0="; };
-  cargoDeps = rustPlatform.fetchCargoTarball { inherit src; name = "${pname}-${version}"; hash = "sha256-W4FyqwJpimf0isQRCq9TegpTQPQfsumx40AFQCFG5VQ="; };
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    group = "World";
+    owner = "design";
+    repo = "contrast";
+    rev = version;
+    sha256 = "cypSbqLwSmauOoWOuppWpF3hvrxiqmkLspxAWzvlUC0=";
+  };
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-W4FyqwJpimf0isQRCq9TegpTQPQfsumx40AFQCFG5VQ=";
+  };
   nativeBuildInputs = [
@@ -25,3 +39,10 @@
   '';
-  meta = with lib; { description = "Checks whether the contrast between two colors meet the WCAG requirements"; homepage = "https://gitlab.gnome.org/World/design/contrast"; license = licenses.gpl3Plus; maintainers = with maintainers; [ jtojnar ]; platforms = platforms.unix; };
+  meta = with lib; {
+    description =
+      "Checks whether the contrast between two colors meet the WCAG requirements";
+    homepage = "https://gitlab.gnome.org/World/design/contrast";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ jtojnar ];
+    platforms = platforms.unix;
+  };
 }
