diff --git a/idioms/pkgs/hello/after-nixpkgs-fmt.nix b/idioms/pkgs/hello/after-alejandra.nix
index ce49cfd..dd0a2b5 100644
--- a/idioms/pkgs/hello/after-nixpkgs-fmt.nix
+++ b/idioms/pkgs/hello/after-alejandra.nix
@@ -1,8 +1,9 @@
-{ lib
-, stdenv
-, fetchurl
-, nixos
-, testVersion
-, testEqualDerivation
-, hello
+{
+  lib,
+  stdenv,
+  fetchurl,
+  nixos,
+  testVersion,
+  testEqualDerivation,
+  hello,
 }:
@@ -21,3 +21,3 @@ stdenv.mkDerivation rec {
   passthru.tests = {
-    version = testVersion { package = hello; };
+    version = testVersion {package = hello;};
 
@@ -25,5 +25,5 @@ stdenv.mkDerivation rec {
       testEqualDerivation
-        "hello must not be rebuilt when environment.noXlibs is set."
-        hello
-        (nixos { environment.noXlibs = true; }).pkgs.hello;
+      "hello must not be rebuilt when environment.noXlibs is set."
+      hello
+      (nixos {environment.noXlibs = true;}).pkgs.hello;
   };
@@ -39,3 +39,3 @@ stdenv.mkDerivation rec {
     license = licenses.gpl3Plus;
-    maintainers = [ maintainers.eelco ];
+    maintainers = [maintainers.eelco];
     platforms = platforms.all;
