diff --git a/idioms/pkgs/hello/after-alejandra.nix b/idioms/pkgs/hello/after-nixpkgs-fmt.nix
index dd0a2b5..ce49cfd 100644
--- a/idioms/pkgs/hello/after-alejandra.nix
+++ b/idioms/pkgs/hello/after-nixpkgs-fmt.nix
@@ -1,12 +1,12 @@
-{
-  lib,
-  stdenv,
-  fetchurl,
-  nixos,
-  testVersion,
-  testEqualDerivation,
-  hello,
+{ lib
+, stdenv
+, fetchurl
+, nixos
+, testVersion
+, testEqualDerivation
+, hello
 }:
+
 stdenv.mkDerivation rec {
   pname = "hello";
   version = "2.12";
@@ -19,13 +19,13 @@ stdenv.mkDerivation rec {
   doCheck = true;
 
   passthru.tests = {
-    version = testVersion {package = hello;};
+    version = testVersion { package = hello; };
 
     invariant-under-noXlibs =
       testEqualDerivation
-      "hello must not be rebuilt when environment.noXlibs is set."
-      hello
-      (nixos {environment.noXlibs = true;}).pkgs.hello;
+        "hello must not be rebuilt when environment.noXlibs is set."
+        hello
+        (nixos { environment.noXlibs = true; }).pkgs.hello;
   };
 
   meta = with lib; {
@@ -37,7 +37,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.gnu.org/software/hello/manual/";
     changelog = "https://git.savannah.gnu.org/cgit/hello.git/plain/NEWS?h=v${version}";
     license = licenses.gpl3Plus;
-    maintainers = [maintainers.eelco];
+    maintainers = [ maintainers.eelco ];
     platforms = platforms.all;
   };
 }
