diff --git a/monsters/poor-indentation/after-nixfmt.nix b/monsters/poor-indentation/after-nixpkgs-fmt.nix
index 09f521f..f87c238 100644
--- a/monsters/poor-indentation/after-nixfmt.nix
+++ b/monsters/poor-indentation/after-nixpkgs-fmt.nix
@@ -5,8 +5,8 @@
 
-       Type: concat :: [a] -> [a] -> [a]
+                           Type: concat :: [a] -> [a] -> [a]
 
-       Example:
-         concat [ 1 2 ] [ 3 4 ]
-         => [ 1 2 3 4 ]
-    */
+                           Example:
+                             concat [ 1 2 ] [ 3 4 ]
+                             => [ 1 2 3 4 ]
+                        */
     concat = x: y: x ++ y;
