diff --git a/monsters/poor-indentation/after-alejandra.nix b/monsters/poor-indentation/after-nixpkgs-fmt.nix
index 9b25f69..f87c238 100644
--- a/monsters/poor-indentation/after-alejandra.nix
+++ b/monsters/poor-indentation/after-nixpkgs-fmt.nix
@@ -2,11 +2,11 @@
   lib = {
-    /*
-     Concatenate two lists
 
-     Type: concat :: [a] -> [a] -> [a]
+    /* Concatenate two lists
 
-     Example:
-       concat [ 1 2 ] [ 3 4 ]
-       => [ 1 2 3 4 ]
-     */
+                           Type: concat :: [a] -> [a] -> [a]
+
+                           Example:
+                             concat [ 1 2 ] [ 3 4 ]
+                             => [ 1 2 3 4 ]
+                        */
     concat = x: y: x ++ y;
@@ -16,3 +17,3 @@
     boot.kernel.features = mkOption {
-      default = {};
+      default = { };
       example = literalExpression "{ debug = true; }";
